internal/pkg/remote/endpoint/service.go

```diff
@@ -28,18 +28,18 @@ const defaultTimeout = 10 * time.Second
 // Default Sylabs cloud service endpoints.
 const (
 	SCSDefaultCloudURI     = "cloud.sylabs.io"
-	SCSDefaultLibraryURI   = "https://library.sylabs.io"
+	DefaultApptainerHost   = "ghcr.io"
+	DefaultRegistryURI     = "https://ghcr.io"
```

Comment:
OCI registry URIs already default to `index.docker.io`, so we cant inject our own. The library/registry service concept within the remote endpoint command group should be removed

Reply:
these are injected when not otherwise specified.   specifically, the e2e tests inject empty strings and expect the values in the new diff above.

deferred to #71 and #57